.castor/infra.php

```diff
@@ -99,7 +99,7 @@ function destroy(
     fs()->remove($files);
 }
 
-#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)')]
+#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)', aliases: ['generate-certificates'])]
```

Comment:
this one is useless IHMO, we never use it directly. It can be a drawbacks to add too much aliases because it add more alternative to shortcut, and so conflict